src/pages/landing/stocks/_headerSection.js

```diff
@@ -0,0 +1,384 @@
+    }
+    @media ${device.mobileL} {
+        width: 90%;
+    }
+`
+const SectionImgWrapper = styled(Flex)`
+    width: 45%;
+    @media ${device.laptopM} {
+        align-items: center;
+    }
+    @media ${device.tabletL} {
+        width: 100%;
+        justify-content: center;
+    }
+`
+const SectionImgDiv = styled.div`
+    width: 100%;
+    position: relative;
+    z-index: 3;
+    @media ${device.tabletL} {
+        width: 100%;
+        max-width: 58.8rem;
+    }
+`
+const SectionBannerImg = styled.img`
+    width: 100%;
+    position: absolute;
+    top: -15%;
+    left: 0;
+    @media ${device.laptopM} {
+        position: unset;
+    }
+    @media ${device.tabletL} {
+        margin: 0 auto;
+    }
+`
+const SectionTextWrapper = styled(Flex)`
+    width: 55%;
+    align-items: center;
+    flex-direction: column;
+    margin-left: 120px;
+    @media ${device.laptopM} {
+        margin-left: 80px;
+    }
+    @media ${device.laptop} {
+        margin-left: 60px;
+    }
+    @media ${device.tabletL} {
+        width: 100%;
+        max-width: 58.8rem;
+        margin: 0 auto;
+    }
+`
+const SectionStyledHeader = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 6.4rem;
+        max-width: 60rem;
+        text-align: center;
+    }
+    @media ${device.tabletS} {
+        font-size: 4rem;
+        max-width: 60rem;
+        margin-bottom: 4px;
+    }
+`
+const SectionStyledHeaderSmall = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 2.4rem;
+        max-width: 100%;
+        text-align: center;
+    }
+    @media ${device.tabletS} {
+        font-size: 1.6rem;
+        max-width: 100%;
+    }
+`
+const is_ppc = true
+
+const HeaderSection = () => {
+    const data = useStaticQuery(query)
+
+    return (
+        <MainWrapper>
+            <HeaderWrapper>
+                <TopHeaderWrapper>
+                    <TopHeaderTextDiv>
+                        <TopTextWrapper>
+                            <StyledHeader color="white" type="main-landing-title" mb="2.4rem">
+                                {localize('Stocks, unlocked')}
+                            </StyledHeader>
+                            <StyledHeaderSmall
+                                color="white"
+                                type="sub-section-title"
+                                weight="normal"
+                                size="2.4rem"
+                                mb="4.0rem"
+                            >
+                                {localize(
+                                    'Trade global stocks and stock indices now on our DMT5 platform',
+                                )}
+                            </StyledHeaderSmall>
+                            <BtnDiv>
+                                <StyledLinkButton
+                                    type="submit"
+                                    secondary="true"
+                                    to={is_ppc ? '/landing/signup/' : '/signup/'}
```

Comment:
if `is_ppc` is always true this should be set to `'/landing/signup/'` instead of additional checks here

Reply:
link is set to only '/landing/signup/' already..